app/blog-content.css

```
@@ -286,6 +315,41 @@
+      );
+    }
+
+    .dark .blog-content table::after {
+      background: linear-gradient(
+        to left,
+        rgb(10 12 20 / 0.92),
+        rgb(10 12 20 / 0)
+      );
+    }
+
+    .blog-content td > :not(code):not(pre) {
```

Stylelint (17.3.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)